client/dashboard/domains/domain-contact-details/index.tsx

```diff
@@ -47,45 +49,91 @@ export default function DomainContactInfo() {
 		return { initialData, key: JSON.stringify( initialData ) };
 	}, [ registrantWhoisData ] );
 
-	const validateMutation = useMutation( domainWhoisValidateMutation( [ domainName ] ) );
-	const updateMutation = useMutation( domainWhoisMutation( domainName ) );
+	const validateMutation = useMutation( {
+		...domainWhoisValidateMutation( [ domainName ] ),
+		meta: { snackbar: { error: { source: 'server' } } },
+	} );
+	const updateMutation = useMutation( {
+		...domainWhoisMutation( domainName ),
+		meta: {
+			snackbar: {
+				/* translators: %s is the domain name */
+				success: sprintf( __( 'Contact details for %s saved.' ), domainName ),
+				error: { source: 'server' },
+			},
+		},
+	} );
+	const savePrivacyMutation = useMutation( {
+		...domainPrivacySaveMutation( domainName ),
+		meta: {
+			snackbar: {
+				/* translators: %s is the domain name */
+				success: sprintf( __( 'Privacy has been successfully updated for %s!' ), domainName ),
+				error: { source: 'server' },
+			},
+		},
+	} );
+
+	const disclosePrivacyMutation = useMutation( {
+		...domainPrivacyDiscloseSaveMutation( domainName ),
+		meta: {
+			snackbar: {
+				success: sprintf(
+					/* translators: %s is the domain name */
+					__( 'Your contact information for %s is now publicly visible!' ),
+					domainName
+				),
+				error: { source: 'server' },
+			},
+		},
+	} );
+	const redactPrivacyMutation = useMutation( {
```

Comment:
This seems exactly the same as `disclosePrivacyMutation` above 🤔

Reply:
different snackbar message